README.md

```diff
@@ -34,14 +34,25 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
-2. Name two big differences between ```display: block;``` and ```display: inline;```.
+It is a more descriptive and purposeful way of annotating your HTML. You can just use standard markups such as "p,div, and span", but when you use tags such as "h1 and address" you give a clearer meaning to what your webpages content is conveying.
+
+2. Name two big differences between ```display: block;``` and ```display: inline;```
+
+(display: block;) puts content into a "block" with no other elements being able to be beside it by default. (display: inline;) puts other elements on the same line as it, by default.
```

Comment:
This is close, but I don't believe `inline` automatically puts elements on the same line, unless they are _all_ `inline`, and even then it might wrap or such. The positioning effects are caused the fundamental properties of the two:

* Block elements take up the whole with of the container they're in. That means any elements would be forced to be under it, creating a column.
* Inline elements take up only the amount of width needed to fit its content. That means it is _possible_ to place other inline elements beside it to create a row.